<launch>
    <param name="cmd_vel_topic" type="string" value="cmd_vel"/>
    <node pkg="sentry_serial" type="sentry_send" name="sentry_send" output="screen" 
    args="/dev/ttyUSB0">

    <!-- 
        虚拟串口-ttyACM0
        usb转ttl-ttyUSB0
     -->
	
    </node>


</launch>